### PR TITLE
MIPS module refactor

### PR DESCRIPTION
**Pull request by aethelwulffe**
Move all assets used by QA reporting to an integral module.  Begin
update to 2018 reporting year standards.

### CODE REVIEW

**Comment by teryhill** [comment]
I tested this with the exception of the 2018 files during the last month and a half.

**Comment by aethelwulffe** [comment]
Please pay close attention to files that I touched that are NOT the mips module, and were NOT in library/classes/

File: modules/MIPS/SQL/tests/PQRS_settings_config.sql [hunk 1]
```diff
@@ -240,7 +240,7 @@ REPLACE INTO `globals` (`gl_name`, `gl_index`, `gl_value`) VALUES
 ('enable_scanner', 0, ''),
 ('scanner_output_directory', 0, '/mnt/scan_docs'),
 ('portal_onsite_enable', 0, ''),
-('portal_onsite_address', 0, 'https://your_web_site.com/openemr/patient_portal'),
+('portal_onsite_address', 0, 'https://your_web_site.com/openemr/patients'),
```
Comment (teryhill) [inline modules/MIPS/SQL/tests/PQRS_settings_config.sql]: I think the line you deleted was more correct

File: modules/MIPS/SQL/tests/PQRS_settings_config.sql [hunk 2]
```diff
@@ -420,7 +420,7 @@ INSERT INTO `layout_options` (`form_id`, `field_id`, `group_name`, `title`, `seq
 ('LBTphreq', 'body', '1', 'Details', 10, 3, 2, 30, 0, '', 1, 3, '', '', 'Content', 5, '', 'F', ''),
 ('LBTlegal', 'body', '1', 'Details', 10, 3, 2, 30, 0, '', 1, 3, '', '', 'Content', 5, '', 'F', ''),
 ('LBTbill', 'body', '1', 'Details', 10, 3, 2, 30, 0, '', 1, 3, '', '', 'Content', 5, '', 'F', ''),
-('HIS', 'risk_factors', '1General', 'Risk Factors', 1, 21, 1, 0, 0, 'riskfactors', 1, 1, '', '', 'Risk Factors', 0, '', 'F', ''),
+('HIS', 'usertext11', '1General', 'Risk Factors', 1, 21, 1, 0, 0, 'riskfactors', 1, 1, '', '', 'Risk Factors', 0, '', 'F', ''),
```
Comment (teryhill) [inline modules/MIPS/SQL/tests/PQRS_settings_config.sql]: I thought we got rid of usertext11

File: modules/MIPS/ajax/collect_new_report_id.php [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package LibreEHR
+ * @author  Brady Miller <brady@sparmy.com>
```
Comment (teryhill) [inline modules/MIPS/ajax/collect_new_report_id.php]: Did he write this?

**Comment by aethelwulffe** [comment]
Updated, tested, in-production.

**Comment by aethelwulffe** [comment]
Well, seeing as we have commits elsewhere against outdated code, this is in production, and everything is in one module...and review/testing is unlikely elsewhere, I am merging this to keep ongoing stuff in sync.  This should be an optional module anyway.

**Comment by teryhill** [comment]
Go fer it

**Pull request merged**